Algorithmia/datafile.py

```diff
@@ -14,6 +15,8 @@ class DataFile(DataObject):
     def __init__(self, client, dataUrl):
         super(DataFile, self).__init__(DataObjectType.file)
         self.client = client
+        # determine whether file is local or in Algorithmia data://
+        self.is_local = not dataUrl.startswith('data://')
```

Comment:
this will fail for other data connector paths like `s3://` i would suggest maybe going with requiring something like `file://` as a prefix for local files? idk

Reply:
yeah `file://` is the idiomatic prefix, we've also used this in some other projects that allow for local file utilization.

Reply:
I was wondering about that too, but doing that would remove the ability to have relative paths and force people to have absolute ones.  You can see for example in the unit tests that I use relative paths.

Reply:
@pmcq Do we allow all of those other prefixes?  In that case it might make sense to refactor this and not allow relative file paths

Reply:
@pmcq @dmorris-algo @zeryx It seems there's a consensus to have this stuff start with "file://".  I realized that we can still support local relative and absolute paths IF we say that everything after the "file://" counts as the file path.  So for example (count the slashes) file://myfile.txt is relative but file:///path/to/myfile.txt is absolute.  I worry about confusion with lots of "/" characters but this seems to me like the most sensible option